egg.json

```diff
@@ -10,6 +10,7 @@
     "./create_app.ts",
     "./src/**/*",
     "./README.md",
+    "./mod.ts",
```

Comment:
mod.ts twice?

Reply:
oh yeah.. huh :L good spot ill remove it

Reply:
https://github.com/drashland/deno-drash/pull/388/commits/c2b0623cd0d1a38ce62e87bbe5f69f0c77452df2